frontend/src/edit.jsx

```diff
@@ -83,15 +89,25 @@ export default function EditMenu(props) {
       currentScenarioID = await makeScenario(scenarioName, 'description');
       setScenarioID(currentScenarioID);
     }
-    const jid = await doWallpaper(parcel.coords, pattern, currentScenarioID);
+    let jid;
+    if (conversionOption === 'wallpaper' && selectedPattern) {
+      jid = await doWallpaper(parcel.coords, selectedPattern, currentScenarioID);
+    }
+    if (conversionOption === 'paint' && singleLULC) {
+      jid = await convertToSingleLULC(parcel.coords, singleLULC, currentScenarioID);
+    }
     setJobID(jid);
     refreshSavedScenarios();
   }
 
   // TODO: do handlers  need to be wrapped in useCallback? 
   // to memoize the function?
-  const handleRadio = (event) => {
-    setPattern(event.target.value);
+  const handleSelectLULC = (event) => {
+    setSingleLULC(event.target.value);
+  };
+
+  const handleConversionOption = (event) => {
+    setConversionOption(event.target.value);
```

Comment:
Since these are one liners I might put them inline in the JSX like we have for the scenario name input

Reply:
Good idea, I did this. Technically, if they are defined in the JSX then a new function gets created for each one on each render. But that performance hit is negligible.